README.md

```diff
@@ -372,227 +535,198 @@ let result = try await Subprocess.run(
+/// subprocess nor waiting until it exits.
+///
+/// This method is useful for launching subprocesses that outlive their
+/// parents (for example, daemons and trampolines).
+///
+/// - Parameters:
+///   - configuration: The `Subprocess` configuration to run.
+///   - input: A file descriptor to bind to the subprocess' standard input.
+///   - output: A file descriptor to bind to the subprocess' standard output.
+///   - error: A file descriptor to bind to the subprocess' standard error.
+/// - Returns: the process identifier for the subprocess.
+@available(macOS 9999, *)
+public func runDetached(
+    _ configuration: Configuration,
+    input: FileDescriptor? = nil,
+    output: FileDescriptor? = nil,
+    error: FileDescriptor? = nil
+) throws -> ProcessIdentifier
 ```
 
 
-### `Subprocess.Execution`
+### `Execution`
 
-In contrast to the monolithic `Process`, `Subprocess` utilizes two types to model the lifetime of a process. `Subprocess.Configuration` (discussed later) and `Subproces.Execution`. `Subprocess.Execution` is designed to represent an executed process. This execution could be either in progress or completed. Direct construction of `Subprocess.Execution` instances is not supported; instead, a `Subprocess.Execution` object is passed to the `body` closure of `run()`. This object is only valid within the scope of the closure, and developers may use it to send signals to the child process or retrieve the child's standard I/Os via `AsyncSequence`s.
+In contrast to the monolithic `Process`, `Subprocess` utilizes two types to model the lifetime of a process. `Configuration` (discussed later) and `Execution`. `Execution` is designed to represent an executed process. This execution could be either in progress or completed. Direct construction of `Execution` instances is not supported; instead, a `Execution` object is passed to the `body` closure of `run()`. This object is only valid within the scope of the closure, and developers may use it to send signals to the child process or retrieve the child's standard I/Os via `AsyncSequence`s.
 
 ```swift
-extension Subprocess {
-    /// An object that repersents a subprocess that has been
-    /// executed. You can use this object to send signals to the
-    /// child process as well as stream its output and error.
-    @available(FoundationPreview 0.4, *)
-    @available(iOS, unavailable)
-    @available(tvOS, unavailable)
-    @available(watchOS, unavailable)
-    public struct Execution<
-        Input: Subprocess.InputProtocol,
-        Output: Subprocess.OutputProtocol,
-        Error: Subprocess.OutputProtocol
-    >: Sendable {
-        /// The process identifier of the current execution
-        public let processIdentifier: ProcessIdentifier
-    }
+/// An object that repersents a subprocess that has been
+/// executed. You can use this object to send signals to the
+/// child process as well as stream its output and error.
+@available(macOS 9999, *)
+public struct Execution<
```

Comment:
If this is only valid in the scope of the closure, does it make sense to mark it as `~Escapable`?